media-proxy/src/mesh/conn.cc

```diff
@@ -273,6 +274,7 @@ const char * result2str(Result res)
     case Result::error_bad_argument:     return "bad argument";
     case Result::error_out_of_memory:    return "out of memory";
     case Result::error_general_failure:  return "general failure";
+    case Result::error_shutdown:         return "processing shutdown";
```

Comment:
```suggestion
    case Result::error_context_cancelled: return "context cancelled";
```

Reply:
Fixed in 7b57d140d1ff8be6328bad2c1e5c18c53ad97bae